### ISSUE TITLE
Using "queue" with videos stops between jobs asking if the video has to be buffered

### ISSUE THREAD

**Issue by fabriziocarraro** [usc-imi/aeo-light#35]
When using a video as a source (on mac) aeo-light asks if the video has to be buffered, (this in some cases increases the speed performances). This is ok for single jobs, but it is a problem when adding many jobs in the queue, because the queue stops at each job asking if I want to buffer the next video. Is there a way to set as a default that videos have to be buffered (or not buffered)? It does not make sense to have a queue that stops between jobs, isn't it?

**Comment by sjohnson-imi** [comment]
Good point. Probably the buffered/unbuffered question is best left to a preference rather than a question for each new source change. I'll add that to the to-do list. Thanks.
